front/app/containers/App/constants.js

```diff
@@ -170,6 +174,7 @@ exports.appLocalesMomentPairs = {
   'ro-RO': 'ro',
   'pt-BR': 'pt',
   'ar-SA': 'ar',
+  'ar-MA': 'ar-ma',
```

Comment:
When I tried using `ma` here (again going against our convention of using the first pair of letters, not second pair), I got this error in CI `front-build`:
```
[webpack-cli] Failed to load '/citizenlab/front/internals/webpack/webpack.config.js' config
[webpack-cli] Error: MomentLocalesPlugin: Moment.js doesn’t include a locale you specified: ma. Check the plugin’s `localesToKeep` option.
You can see the full list of locales that Moment.js includes in node_modules/moment/locale/.
If you would like unsupported locales to be ignored, please use the `ignoreInvalidLocales` option.
    at normalizeLocalesToKeep (/citizenlab/front/node_modules/moment-locales-webpack-plugin/index.js:44:15)
    at checkOptions (/citizenlab/front/node_modules/moment-locales-webpack-plugin/index.js:21:25)
    at new MomentLocalesPlugin (/citizenlab/front/node_modules/moment-locales-webpack-plugin/index.js:103:29)
    at Object.<anonymous> (/citizenlab/front/internals/webpack/webpack.config.js:200:7)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)

Exited with code exit status 2
```
I noticed that we do sometimes use a lowercase double pair (e.g: `en-gb`), so tried `ar-ma` and CI passed.

Reply:
Yes, I forgot to say I also checked the moment list, from the error message.